eslint.config.js

```diff
@@ -0,0 +1,27 @@
+const globals = require("globals");
+const js = require('@eslint/js');
+const cds = require('@sap/eslint-plugin-cds');
+
+module.exports = [
```

Comment:
Don't we need a section for our plugin as well, similar to what we now generate for new projects?

Reply:
solved